app/models/manager_refresh/inventory_collection_default/cloud_manager.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
Why is this being added in this PR?  It's unrelated and can introduce unforseen issues.